Directory.Packages.props

```diff
@@ -62,15 +62,15 @@
     <PackageVersion Include="Microsoft.Health.Api" Version="$(HealthcareSharedPackageVersion)" />
     <PackageVersion Include="Microsoft.Health.Client" Version="$(HealthcareSharedPackageVersion)" />
     <PackageVersion Include="Microsoft.Health.Core" Version="$(HealthcareSharedPackageVersion)" />
-    <PackageVersion Include="Microsoft.Health.Extensions.BuildTimeCodeGenerator" Version="$(HealthcareSharedPackageVersion)" />
+    <PackageVersion Include="Microsoft.Health.Extensions.BuildTimeCodeGenerator" Version="6.1.129" />
```

Comment:
Any reason to not keep a consistent version from HealthcareSharedPackageVersion?

Reply:
Sorry I notice that the HealthcareSharedPackageVersion has updated to 6.1.129, so I changed the code and restored this file.